app/scripts/views/coppa/coppa-date-picker.js

```diff
@@ -90,14 +90,17 @@ define([
       if (! this._validateYear()) {
         this.addInvalidRow('#fxa-age-year');
 
-        this.showValidationError('#fxa-age-year',
-                AuthErrors.toError('YEAR_OF_BIRTH_REQUIRED'));
+        var yearError = AuthErrors.toError('YEAR_OF_BIRTH_REQUIRED');
+        this.showValidationError('#fxa-age-year', yearError);
+        this.logError(yearError);
```

Comment:
@vladikoff - I'm really confused why the `logError` lines are needed, that's the first thing done in [showValidationError](https://github.com/mozilla/fxa-content-server/blob/46484479951e4d784b4d517a6f520494692c760d/app/scripts/views/form.js#L422)